tmd/io/conversion.py

```diff
@@ -48,9 +52,14 @@ def _section_to_data(section, tree_length, start, parent):
     parents = np.arange(tree_length - 1, tree_length + n - 1, dtype=np.int64)
     parents[0] = parent
 
+    try:
+        diameters = section.diameters[start:]
+    except AttributeError:
+        diameters = np.zeros(n, dtype=points.dtype)
+
```

Comment:
Is this really possible? I am pretty sure morphio would throw if diameters are not available when reading the morphology. Isn't that right?

Reply:
If I remember correctly I wanted to make it work with NeuroM sections which do not have a diameter attribute. But I guess we could just try to convert the section and then process it normally.

Reply:
These conversion functions are specific to morphio and should remain as such. Adding implicit knowledge of how neurom lays out its data, binds these functions to the public API of neurom in addition to MorphIO's one.

In neurom v4, the morphio morphology will be stored as an attribute (instead of inheriting from it) and the public API will change quite a bit towards a cleaner separation between the holder morphology and the neurom class.

Getting the morphio object from a neurom morphology will be sth like:
```python
morphio_morphology = neurom_morphology.to_morphio()
```

Reply:
Yeah ok.
So when we want to use a NeuroM v3 morph with TMD we have to convert it into a MorphIO morph before.
And when is Morphio v4 scheduled?